app/assets/words/polish_words.txt

```diff
@@ -0,0 +1,99 @@
+i, and  
```

Comment:
No need for the English translation as mentioned in the issue, we will use a translation package.
Will alow it for now but will be changed soon.